fun/draw.cpp

```diff
@@ -36,10 +32,16 @@ int setupTerm() {
   if (tcsetattr(0, TCSAFLUSH, &newtermios) < 0) {
     return -1;
   }
+
+  // Switch to alternate screen mode.
+  printf("\033[?1049h");
```

Comment:
It's also how full screen text apps tend to work, and simplifies a bunch of stuff. I bet you can even skip the "clear" come to think of it.